lib/services/lwm2mHandlers/commonLwm2m.js

```diff
@@ -88,24 +86,28 @@ function observeActiveAttributes(payload, registeredDevice, callback) {
     }
 
     for (var i = 0; i < activeAttributes.length; i++) {
-        var lwm2mMapping = {};
+        // Ezequiel: esto debe ser nulo, y todos los if han sido modificados
```

Comment:
I don't remember, it may be the result of some tests, which had no result. I'll let it be as before as the change.